src/scheduler.js

```diff
@@ -86,19 +86,25 @@ function phaseRunner(gatherers) {
   };
 }
 
+// This isn't exposed functionality, and it's not core functionality, so skipped. */
+/* istanbul ignore next */
 function saveArtifacts(artifacts) {
```

Comment:
if we are not using the code we should delete it with git it is very easy to bring back